src/App.js

```diff
@@ -32,6 +33,7 @@ function App() {
       <h1>Saviors and Starters</h1>
       <Switch>
         <Route path='/ability_scores/:stat'>{<StatPage stat={stat}/>}</Route>
+        <Route path='/skills/:skill'>{<SkillPage skill={skill}/>}</Route>
         <Route path='/'>{<Homepage data={testData} search={handleSearch} />}</Route>
```

Comment:
Nice, love to see a Switch used! I am sure you plan to add this in the future, but I would recommend adding an error page/component for any other url like this: `<Route path='*'> <ErrorPage /> </Route>`